### DISCUSSION TITLE
FR: Shopify Collections Image

### DISCUSSION THREAD

**Discussion by austriker27** [statamic-rad-pack/shopify#125] [Feature Requests]
# IDEA 
It would be super nice if when the shopify collections were imported, the string/url of the "collection image" would be included. (I did set this up in Statamic but I think its best to rely on Shopify as much as possible for the product content)

# SCREENSHOT:
![image](https://user-images.githubusercontent.com/14855925/131413547-e6551a0c-9e13-41bc-8b99-55a8ee88015b.png)

**Comment by austriker27** [comment]
PS: Also it would be real handy if it also imported the collections description!

**Comment by ryanmitchell** [comment]
This will be included in the next release.
